dhis-2/dhis-services/dhis-service-reporting/src/test/java/org/hisp/dhis/predictor/PredictionAnalyticsDataFetcherTest.java

```diff
@@ -179,7 +179,7 @@ void testGetValues()
             .addValue( programIndicatorB.getUid() )
             .addValue( orgUnitA.getUid() )
             .addValue( aocC.getUid() )
-            .addValue( 20.0 );
+            .addValue( 20 ); // Note: boxed as Long, not Double
```

Comment:
If this should be a `Long` it needs to be `20L`, `20` is an `Integer`. For the sake of the test this might not make a difference but the comment becomes confusing.

Reply:
Nice catch, fixed!